core/common/src/main/java/alluxio/PropertyKey.java

```diff
@@ -19,8 +19,14 @@
  * you change or add Alluxio configuration properties.
  */
 public enum PropertyKey {
+  CONF_DIRECTORY(Name.CONF_DIRECTORY, "${alluxio.home}/conf"),
   DEBUG(Name.DEBUG, false),
+  /**
+   * @deprecated use INSTALL_HOME or WORK_DIRECTORY instead
```

Comment:
NIT on javadoc: should either deprecated by "ALLUXIO_INSTALL_HOME" or "alluxio.install.home"?